release_images.yml

```diff
@@ -109,6 +109,30 @@ release_images:
       disable_sm_tag: False  # [Default: False] This option is not used by Example images
       force_release: False
   9:
+    framework: "tensorflow"
```

Comment:
You need to remove the entry for cu102, or change it to cu110, because they cannot both exist in the release_images.yml file at the same time.